lib/matplotlib/backends/backend_qt5agg.py

```diff
@@ -71,7 +71,11 @@ def paintEvent(self, e):
         In Qt, all drawing should be done inside of here when a widget is
         shown onscreen.
         """
-        FigureCanvasAgg.draw(self)
+        if not hasattr(self, "_agg_redraw_flag"):
```

Comment:
It should, but this is seems to be a Mixin for which the init method is not called at all.